test/e2e/upgrade_test.go

```diff
@@ -233,15 +234,19 @@ var _ = Describe("BMO Upgrade", func() {
 		}
 
 		if e2eConfig.GetVariable("UPGRADE_DEPLOY_BMO") != "false" {
-			By("Installing BMO on the upgrade cluster")
+			bmoKustomization := e2eConfig.GetVariable("UPGRADE_BMO_KUSTOMIZATION_FROM")
+			// Get the index of the last "release" word, which typically indicates the start of the BMO version name
+			lastReleaseWordIndex := strings.LastIndex(bmoKustomization, "release")
+			BmoFromVer := bmoKustomization[lastReleaseWordIndex:]
+			By(fmt.Sprintf("Installing BMO with version: %s on the upgrade cluster", BmoFromVer))
```

Comment:
This basically adds a requirement on the kustomization folder name. I would rather have something simpler that doesn't care what you name them. Perhaps just print the path?
```suggestion
			bmoKustomization := e2eConfig.GetVariable("UPGRADE_BMO_KUSTOMIZATION_FROM")
			By(fmt.Sprintf("Installing BMO from %s on the upgrade cluster", bmoKustomization))
```

Reply:
Hmm. Okay. How about using the Base path instead? Having a full path in the log is fine, but I also want to have a nice way to notice it in the artifacts.

Reply:
Oh I didn't notice it was used for the artifacts. Thanks for pointing it out!
I think the base path sounds good for artifacts, in the log I would say the more we include the better :D

Reply:
Sounds good. I'll update it real quick